Gruntfile.js

```diff
@@ -2,6 +2,9 @@ module.exports = function(grunt) {
   var pkg = grunt.file.readJSON('package.json');
   var betaVersion = pkg.version.toString() + "." + (Math.floor(Date.now() / 60000) - 24365210);
 
+  grunt.loadNpmTasks('grunt-contrib-uglify');
+  grunt.loadNpmTasks('grunt-contrib-concat');
```

Comment:
Would you mind putting the grunt changes into a separate PR? Then I can merge just the XP bar stuff.